gallery/tutorials/plot_lecture_feature_demo.py

```diff
@@ -0,0 +1,460 @@
+"""
+ASPIRE-Python Introduction
+==========================
+
+In this notebook we will introduce some code from ASPIRE-Python
+that corresponds to topics from MATH586.
+"""
+
+# %%
+# Imports
+# -------
+# First we import some of the usual suspects. In addition, we import some classes from
+# the ASPIRE package that we will use throughout this tutorial.
+
+# %%
+# Homework Task 0
+# ^^^^^^^^^^^^^^^
+#
+# Attempt to install ASPIRE on your machine. ASPIRE can generally install on
+# Linux, Mac, and Windows
+# under Anaconda Python, by following the instructions in the README.
+# `The instructions for developers is the most comprehensive
+# <https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/master/README.md#for-developers>`_.
+# Linux is the most tested platform.
+#
+# ASPIRE requires some resources to run, so if you wouldn't run typical data
+# science codes on your machine (maybe a netbook for example),
+# you may use TigerCPU. After logging into TigerCPU,
+# ``module load anaconda3/2020.7`` and follow the anaconda instructions for
+# developers in the link above.
+# Those instructions should create a working environment for tinkering with
+# ASPIRE code found in this notebook.
+
+import logging
+
+import matplotlib.pyplot as plt
+import mrcfile
+import numpy as np
+
+from aspire.abinitio import CLSyncVoting
+from aspire.noise import AnisotropicNoiseEstimator, WhiteNoiseEstimator
+from aspire.operators import FunctionFilter, RadialCTFFilter, ScalarFilter
+from aspire.source import RelionSource, Simulation
+from aspire.utils.coor_trans import (
```

Comment:
These should be deprecated and replaced with methods from `Rotation`. No need to do this here but can you create an issue?